### PR TITLE
Improve Synchronization and Signal APIs documentation

### PR DESCRIPTION
**Pull request by nebilbenmabrouk**

### CODE REVIEW

**Review by Hiba-Alili — APPROVED** [review]

File: src/docs/user/ProActiveUserGuide.adoc [hunk 1]
```diff
@@ -2352,6 +2352,9 @@ To know in which scripts these bindings are available, study the <<_variables_qu
 
 This Synchronization API allows for both communication and synchronization between Tasks and between Jobs dynamically at execution. It is based on a Key-Value Store offering atomic operations. It enables to easily put into action all classical synchronization patterns: Critical Sections and Mutual Exclusion, Rendez-vous, Barriers, Producer/Consumer...
 
+To experience concrete use of the Synchronization API, just go to <<_proactive_studio, ProActive Studio>> and use the predefined templates in the Controls menu (e.g., *Key_Value*).
```
Comment (mboussaa) [inline src/docs/user/ProActiveUserGuide.adoc:2355]: To get famliar with the Synchr... (just a suggestion)

File: src/docs/user/ProActiveUserGuide.adoc [hunk 2]
```diff
@@ -2619,8 +2622,10 @@ A complete example of Producer / Consumer, _ServiceDeployerConsumer_ can be down
 The *Signal API* is a high-level API that enables users to interact with running jobs via signals (we also call them actions).
 That is, it allows a job to expose specific signals (e.g., `soft_stop` to properly finish a job) that can be triggered by users from the *Workflow Execution portal*.
 You can try the <<_examples_2,examples>> below to better understand how it works.
+In summary and for simple use, just go to <<_proactive_studio, ProActive Studio>> and use the 2 predefined templates in the Controls menu: *Check_For_Signals* or *Wait_For_Signals*.
+Fill up the Task Variable *SIGNALS*, and use those templates in your own workflow to be able to receive input from, for instance, the Workflow Execution portal.
 
-At a high-level view, the Signal API allows jobs and users to exchange signals using a communication channel provided by the <<_task_synchronization_api,synchronization API>>.
+At a more detailed level and advanced use, the Signal API allows jobs and users to exchange signals using a communication channel provided by the <<_task_synchronization_api,synchronization API>>.
```
Comment (mboussaa) [inline src/docs/user/ProActiveUserGuide.adoc:2628]: (To make it short but this is also a suggestion) At a more advanced level, the...

**Review by mboussaa — APPROVED** [review]

File: src/docs/user/ProActiveUserGuide.adoc (hunk 2 above)
Comment (nebilbenmabrouk) [reply src/docs/user/ProActiveUserGuide.adoc:2628]: Denis wants it like this :)

Comment (mboussaa) [reply src/docs/user/ProActiveUserGuide.adoc:2628]: Ok np

File: src/docs/user/ProActiveUserGuide.adoc (hunk 1 above)
Comment (nebilbenmabrouk) [reply src/docs/user/ProActiveUserGuide.adoc:2355]: The aim is to let the user have a direct and concrete use of the Synchronization API (without reading the whole documentation), that is why I prefer the terms 'experience' and 'concrete'. Thanks

**Pull request merged**